conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+
+            <p>An Ombudsperson can advise or assist you with:</p>
+
+            <ul>
+                <li>How the Code of Conduct may apply in your situation</li>
+                <li>How to start the mediation process</li>
+                <li>How to raise a formal complaint</li>
+            </ul>
+
+            <p>TODO: More about the role of the ombudsperson and what they can and cannot do.</p>
+        </section>
+
+        <section id="mediation">
+            <h2>Mediation</h2>
+
+            <p>Mediation is most effective when used earlier in a dispute, before people become fixed in their position, as it provides a positive opportunity to resolve practical problems.</p>
+
+            <p>It empowers people to work together to resolve issues, and can therefore be less stressful and have a less negative impact on individuals. Mediation can also be successful in later stages after an investigation has concluded to help people rebuild relationships.</p>
+                
+            <p>It is usually between two parties, but mediation can also work for groups. Mediation can be used to resolve a range of issues including a breakdown in relationships or ongoing conflicts between group members.</p>
+
+            <p>Where it's appropriate to attempt to resolve an issue with mediation, there are two approaches that can be taken:</p>
+
+            <ul>
+                <li>Informal Mediation: This approach may include solely the parties involved in the incident, or extend to trusted individuals to help facilitate the conversation (for example, chairs or ombudspeople). The process may be more informal in that it takes place over a call or email correspondance, but the goal remains: resolution of conflict.</li>
+                <li>Formal Mediation: This approach involves the parties involved, as well as a professional mediator. It may also include parties like ombudspeople or chairs to help facilitate or provide necessary context or support. A professional mediator will be an independent third-party who is brought in to find a resolution to the conflict.</li>
+            </ul>
+
+            <section id="professional-mediation">
+                <h3>Professional Mediation</h3>
+
+                <p>In cases where it's deemed necessary to bring in a professional mediator, all parties involved:</p>
+
+                <ul>
+                    <li>will agree to attend mediation as an option for resolving the issue</li>
+                    <li>will individually attend a meeting with independent trained mediators</li>
+                    <li>will collectively attend a meeting to state how each party sees the problem and how it might be resolved</li>
```

Comment:
I suggest deleting the above two bullets (about individually and collectively attending meetings).  I don't think we should dictate how the professional mediation should be conducted.  Professional mediators should know better than us about how to make the mediation process effective.